### ISSUE TITLE
Weird colours in reconstructed image

### ISSUE THREAD

**Issue by code-Assasin** [junyanz/pytorch-CycleGAN-and-pix2pix#525]
Hey I am trying to train the CycleGan model for normal to foggy weather condition translation on a custom dataset. I ran the following command to train the networks : 

`python train.py --dataroot ./datasets/custom_data  --name  experiement --model cycle_gan --lambda_identity 10 --loadSize 256 --fineSize 256 --batch_size 2 --gpu_ids 3 --norm batch --init_type kaiming --no_dropout --no_flip --display_port 8097`
(Using older commit)

The loss stops decreasing after some time and the reconstructed images seem to have weird colors:

<img width="884" alt="screen shot 2019-02-12 at 5 37 43 pm" src="https://user-images.githubusercontent.com/15107872/52634323-f0493580-2eec-11e9-9e4c-fc77d339e0ac.png">

**Comment by junyanz** [comment]
Could you use the latest code and the default parameters?

**Comment by code-Assasin** [comment]
Issue Resolved
